src/main/java/ua/rent/masters/easystay/service/impl/BookingServiceImpl.java

```diff
@@ -89,9 +90,17 @@ public BookingResponseDto getById(Long bookingId) {
     @Override
     @Transactional
     public BookingResponseDto updateById(
-            Long bookingId, BookingRequestUpdateDto requestUpdateDto) {
+            Long bookingId, BookingRequestUpdateDto requestUpdateDto, Long userId) {
+        User user = userRepository.findById(userId).orElseThrow(
+                () -> new EntityNotFoundException("User does not exist"));
```

Comment:
I believe get here user better option.
```suggestion
            Long bookingId, BookingRequestUpdateDto requestUpdateDto, User user) {
```